README.md

```diff
@@ -1,24 +1,178 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
 
-Things you may want to cover:
+  <h1><b>Dollar Dairy</b></h1>
 
-* Ruby version
+</div>
 
-* System dependencies
+<!-- TABLE OF CONTENTS -->
 
-* Configuration
+# 📗 Table of Contents
 
-* Database creation
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀Live Demo](#href="https://budget-app-render-jjqs.onrender.com/.)
+    - [Video](#href="https://www.loom.com/share/8747306427bb4d94ab932f2dd7c88fbc?sid=cf1b51e3-8804-427e-8889-056a783e906f")
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Database initialization
+<!-- PROJECT DESCRIPTION -->
 
-* How to run the test suite
+# 💳Dollar Dairy <a name="about-project"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+**Dollar Dairy** Your Path to Financial Clarity. Effortlessly manage your finances with a sleek splash screen, intuitive sign-up, and secure login. Explore categories, track transactions, and create a bright financial future. With seamless navigation and smart features like adding new categories and transactions, taking charge of your money has never been more inviting. Tested for quality, built for you – start your financial journey today.
 
-* Deployment instructions
+## 🛠 Built With <a name="built-with"></a>
 
-* ...
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>localhost</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>PostgreSQL</li>
+  </ul>
+</details>
+
+### Deployment
+
+the site is deployed below:
+
+<a href="https://budget-app-render-jjqs.onrender.com/"> Check out the Live Demo here</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Video
+
+<a href="#">Demonstration video</a>
```

Comment:
- According to the project requirements, your documentation should contain a video presentation, the presentation should contain the following:
```
1. The student recorded a video presenting the features of the project.
1. The maximum length of the video is 5 minutes.
1. The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
1. The video contains a demo of the project, highlighting all the project features.
1. The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

> **Note:** It's recommended that students use [Loom](https://www.loom.com/) to record to video but they can use any tool they want as long as they share an online link of the uploaded video.

```